src/Components/Pages/RentCalculator/RentCalculator.tsx

```diff
@@ -0,0 +1,152 @@
+import { ContentBoxFooter } from "../../ContentBox/ContentBox";
+import { Header } from "../../Header/Header";
+import { useAccordionsOpenForPrint } from "../../../hooks/useAccordionsOpenForPrint";
+import { JFCLLinkInternal } from "../../JFCLLink";
+import "./RentCalculator.scss";
+import {
+  CPI,
+  GoodCauseProtections,
+  UniversalProtections,
+} from "../../KYRContent/KYRContent";
+import { formatMoney } from "../../../helpers";
+import { Button, TextInput } from "@justfixnyc/component-library";
+import { useState } from "react";
+import { PhoneNumberCallout } from "../Results/Results";
+import { gtmPush } from "../../../google-tag-manager";
+
+export const RentCalculator: React.FC = () => {
+  useAccordionsOpenForPrint();
+
+  const increase_pct = CPI + 5;
+
+  const [rentInput, setRentInput] = useState("");
+  const [showRentInput, setShowRentInput] = useState(false);
+
+  const handleInputChange: React.ChangeEventHandler<HTMLInputElement> = (e) => {
+    if (showRentInput) {
+      setShowRentInput(false);
+    }
+    const value = e.target.value;
+    setRentInput(value);
+  };
+
+  const handleSubmit = () => {
+    setShowRentInput(true);
+    gtmPush("gce_rent_calculator_submit");
+  };
+
+  return (
+    <div id="rent-calculator-page">
+      <Header
+        showProgressBar={false}
+        title="Calculate your rent increase"
+        subtitle="If you are covered by Good Cause legislation, you have a right to limited rent
+            increases. Use this calculator to determine the allowable rent increase for your apartment under Good Cause."
+      ></Header>
+      <div className="content-section">
+        <div className="content-section__content">
+          <div className="rent-calculator-callout-box">
+            <span className="callout-box__header">
+              Find out how much your landlord can increase your rent
+            </span>
+            <div className="rent-input-container">
+              <TextInput
+                labelText="Enter the total monthly rent for your entire apartment"
+                type="money"
+                id="rent-input"
+                value={rentInput}
+                onChange={handleInputChange}
+                onWheel={(e) => {
+                  // prevents scroll incrementing value
+                  e.currentTarget.blur();
+                  e.stopPropagation();
+                  setTimeout(() => e.currentTarget.focus(), 0);
+                }}
+              />
+              <Button
+                type="submit"
```

Comment:
I noticed that using "enter" on the input doesn't work to trigger the calculation. Not sure why not, or if it matters tho...

Reply:
hm looks like the "submit" buttons not being triggered by an enter key press is an issue across all input components. e.g. address search, phone number submit, rent calculator submit

might make that into a separate PR just so it doesn't block corey.

Reply:
oh good catch, I didn't realize that either. hmm maybe a component library issue then? let me know if I can help on this is you have other stuff on the go.

Reply:
yeah if you could take this one, that would be mighty helpful!